scylla/src/frame/mod.rs

```diff
@@ -77,6 +77,10 @@ impl SerializedRequest {
     pub fn get_data(&self) -> &[u8] {
         &self.data[..]
     }
+
+    pub fn extract_data(self) -> Vec<u8> {
```

Comment:
ah yes, I'll drop the commit, it's a leftover. It will be useful eventually once we migrate to write_all_vectored, but that doesn't seem to be the case in near future

Reply:
done